content/news/XXXX-XX-XX-mixxx-2_5_3-released.md

```diff
@@ -0,0 +1,53 @@
+title: Mixxx 2.5.3 Released
+authors: Evelynne Veys, Jan Claußen
+tags: 2.5.3, release announcement
+comments: yes
+status: draft
+
+#### Mixxx 2.5.3 Release Announcement: Back to School, back to DVS.
+
+We're proud to announce a new stable release of Mixxx: version 2.5.3, this version not only contains updates and fixes for some issues reported by users, but it also contains improvements to the Digital Vinyl System (DVS) support. These improvements are the result of some bright new insights of developers spinning the wheel instead of sunbathing.
+Since these improvements drastically improve the vinyl experience, we have chosen to backport [^1] it to the 2.5 series instead of integrating it in future alpha & beta versions.
+
+Have a nice Mixxx.
+
+[^1]: Back porting is integrating new work into previous releases in order to let those releases profit from the new development.
+
+#### Digital Vinyl System - Timecode System.
+
+An introduction to Digital Vinyl Systems and timecode can be found in [this article on the site of Digital DJ Tips](https://www.digitaldjtips.com/a-beginners-guide-to-digital-vinyl-systems/), and in our own news archive: [How Does Timecode Vinyl Actually Work? (Pt. 1)]({filename}/news/2021-11-21-dvs-internals-pt1.md) and [How Does Timecode Vinyl Actually Work? (Pt. 2)]({filename}/news/2021-12-22-dvs-internals-pt2.md).
+
+To summarize, Mixxx plays the song by following the signal on the timecode media, which gets internally converted to actual timeframes via a lookup table (LUT).
+So the DJ controls the playback of the digital music by manipulating vinyl or CDs.
+
+In order to create a vinyl-like feeling the song needs to follow the signal as closely as possible (low latency, fast responsiveness) for e.g. scratching or backspins. There is no 'standard' signal - each manufacturer uses its own code. The demodulated code from the signal needs to be converted to a usable timecode. This is realized by storing the states of the code in a LUT which is mapped to instants in time in the song. This makes it possible to quickly jump to positions in time during playback.
```

Comment:
We should explain that there are two signals: an audible one to measure velocity (the tone), and the digital one to determine timecode position.  The code must use both of them together to generate smooth and responsive playback.

Reply:
Technically the code doesn't play a role in any of those features. We don't really have to explain the code here at all. It has been done in other posts.

Reply:
Then maybe drop the bit about LUTs etc.  Just say "In order to create a vinyl-like feeling the song needs to follow the signal as closely as possible (low latency, fast responsiveness) for e.g. scratching or backspins. There is no 'standard' signal - each manufacturer uses its own code. The demodulated code from the signal needs to be converted to a usable timecode. "

Reply:
I requested some changes. The new formulations more accurately describe what is happening and why the new PRs make a difference.